webapp/src/util/dateUtils.ts

```diff
@@ -1,8 +1,15 @@
-export const DAY_MS = 24 * 60 * 60 * 1000;
 export const RECORDING_DURATION = 10 * 1000; // 10 seconds
 
+export enum MS {
+    MINUTE = 60 * 1000,
+    HOUR = 60 * MINUTE,
+    DAY = 24 * HOUR,
+    WEEK = 7 * DAY,
+    YEAR = 365 * DAY,
```

Comment:
This should be removed, and any usages examined.  Years have either 364 or 365 days.